myokit/formats/mathml/_parser.py

```diff
@@ -947,3 +951,18 @@ def _parse_name(self, element):
         except Exception as e:
             raise MathMLError('Unable to create Name: ' + str(e), element)
 
+    def _parse_symbol(self, element):
+        """
+        Parses a ``<csymbol>`` element and returns a :class:`myokit.Name`
+        created by the name factory.
+        """
+
+        timeUri = 'http://www.sbml.org/sbml/symbols/time'
```

Comment:
We'll need to find some way to not have sbml (or cellml) specific stuff in the mathml parser